.github/actions/run-backend-tests/action.yml

```diff
@@ -97,6 +100,15 @@ runs:
                   --snapshot-update\
                   --durations=100 --durations-min=1.0
 
+        - name: Run ee/ tests with person-on-events, no snapshot commits
+          if: ${{ inputs.ee == 'true' && inputs.person-on-events == 'true' }}
+          shell: bash
+          run: |
+              ENABLE_ACTOR_ON_EVENTS_TEAMS=all pytest ee \
+                  --splits ${{ inputs.concurrency }} \
+                  --group ${{ inputs.group }} \
+                  --snapshot-update
```

Comment:
What's supposed to be happening here? Not sure how this is preventing snapshots

Reply:
Oh I see, these updates aren't committed so it never has any impact

Reply:
yep exactly